src/com/tkjngine/gametable/monsterwounds/PogMenuRenderListener.java

```diff
@@ -0,0 +1,117 @@
+                    if (null == pogRegistry.get(activePog))
+                    {
+                        MonsterData monsterData = new MonsterData();
+                        PogAdapter pogAdapter = new PogAdapter(monsterData, activePog);
+                        pogRegistry.put(activePog, pogAdapter);
+                    }
+                }
+            });
+
+            menu.addSeparator();
+            menu.add(item);
+        } else {
+            final MonsterData monsterData = pogAdapter.getMonsterData();
+            final JMenu monsterWoundsMenu = new JMenu("Monster Wounds");
+            menu.add(monsterWoundsMenu);
+            JMenuItem item = null;
+
+            item = new JMenuItem("-1 Wound");
+            item.addActionListener(new ActionListener() {
+                public void actionPerformed(final ActionEvent e) {
+                    monsterData.decWounds();
+                }
+            });
+            monsterWoundsMenu.addSeparator();
+            monsterWoundsMenu.add(item);
+
+            item = new JMenuItem("+1 Wound");
+            item.addActionListener(new ActionListener() {
+                public void actionPerformed(final ActionEvent e) {
+                    monsterData.incWounds();
+                }
+            });
+            monsterWoundsMenu.addSeparator();
+            monsterWoundsMenu.add(item);
+
+            item = new JMenuItem("Refill Wounds");
+            item.addActionListener(new ActionListener() {
+                public void actionPerformed(final ActionEvent e) {
+                    monsterData.setWounds(monsterData.getMaxWounds());
+                }
+            });
+            monsterWoundsMenu.addSeparator();
+            monsterWoundsMenu.add(item);
+
+            item = new JMenuItem("Set Wounds");
+            item.addActionListener(new ActionListener() {
+                public void actionPerformed(final ActionEvent e) {
+                    // max wounds dialog
+                    WoundsDialogue dialogue = new WoundsDialogue(monsterData.getWounds(), monsterData.getMaxWounds());
+                    dialogue.setLocationRelativeTo(GametableFrame.getGametableFrame().getGametableCanvas());
+                    dialogue.setVisible(true);
+                    monsterData.setWounds(dialogue.getValue());
+                }
+            });
+            monsterWoundsMenu.addSeparator();
+            monsterWoundsMenu.add(item);
+
+            item = new JMenuItem("Set Max Wounds");
+            item.addActionListener(new ActionListener() {
+                public void actionPerformed(final ActionEvent e) {
+                    // max wounds dialog
+                    MaxWoundsDialogue dialogue = new MaxWoundsDialogue(monsterData.getMaxWounds());
+                    dialogue.setLocationRelativeTo(GametableFrame.getGametableFrame().getGametableCanvas());
+                    dialogue.setVisible(true);
+                    monsterData.setMaxWounds(dialogue.getValue());
```

Comment:
I did wonder about it and was thinking there may be cases where you would need to increase Max Wounds (Character gained a max wound) while keeping current wounds, but I don't think that applies to monsters and generally characters only gain a max wound when they are full.
I'll make the change